wgsl/index.bs

```diff
@@ -447,7 +451,7 @@ to define a [=uniform buffer=] or a [=storage buffer=]. See [[#memory-layouts]].
   <xmp>
     // Runtime Array
     type RTArr = [[stride(16)]] array<vec4<f32>>;
-    [[block]] struct S {
+    [[layout(storage)]] struct S {
       [[offset(0)]] a : f32;
```

Comment:
Would you still need these `[[offset(n)]]` annotations?

What if the author _wanted_ to specify them?

Reply:
We wouldn't *need* them, but we can discuss separately if we still want them to stay.